Cork/Views/Start Page/Start Page.swift

```diff
@@ -158,12 +159,10 @@ struct StartPage: View
             }
         }
         .padding()
-        .onAppear
-        {
-            Task
-            {
-                upgradeablePackages = await getListOfUpgradeablePackages()
-            }
+        .task(priority: .high) {
```

Comment:
I wasn't sure if there was a reason for using `onAppear` with `Task { .. }` inside? This is equivalent to `.task { .. }` for this use case in my mind

Reply:
I prefer the `onAppear` + `Task` approach because it's easier for me to visualize what's happening. It might be equivalent, but when I'm reading the code, it's easier for me to read
`.onAppear 
{
   Task
   {
   }
}`
than to read `.task` and having to spend that additional time realizing that it fires when the view appears.

Reply:
Fair enough! I believe `.task` will also cancel the task when the view disappears, if it hasn't finished, but I'm not convinced that's a situation that's likely to occur.

I'll revert this part of the PR!

Reply:
Thank you very much! When you do, I will merge the PR